.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: coursier/cache-action@v6
```

Comment:
Does this need to be after setup-java to prevent coursier from installing its own JDK?

The default Scala CI setup on Github does not use `coursier/cache-action` but instead has

```
      uses: actions/setup-java@v4
      with:
        java-version: '11'
        distribution: 'temurin'
        cache: 'sbt'
```